pyproject.toml

```diff
@@ -101,6 +101,7 @@ posthog = "^3.0.2"
 rich = "^13.7.0"
 beautifulsoup4 = "^4.12.2"
 pypdf = "^3.11.0"
+gptcache = "^0.1.43"
 tiktoken = { version = "^0.4.0", optional = true }
 youtube-transcript-api = { version = "^0.6.1", optional = true }
 pytube = { version = "^15.0.0", optional = true }
```

Comment:
Can you please also bump the package version?

Reply:
done